docs/reference/run.md

```diff
@@ -908,7 +908,7 @@ container's logging driver. The following options are supported:
 
 	The `docker logs`command is available only for the `json-file` logging
 driver.  For detailed information on working with logging drivers, see
-[Configure a logging driver](reference/logging/overview.md).
+[Configure a logging driver](/reference/logging/overview).
```

Comment:
I can change this one.

There are a lot of links without a trailing slash out there. Would it help to change those? I do not have any insight into how the server works. What can I do to see what is happening behind the scenes? Is there debug/verbosity flag for the hugo server in `make docs`?